app/[contentType]/[slug]/content.tsx

```diff
@@ -0,0 +1,18 @@
+'use client';
+import hljs from 'highlight.js/lib/core';
+import javascript from 'highlight.js/lib/languages/javascript';
+import { useEffect } from 'react';
+
+export default function Content({ html }: { html: string }) {
```

Comment:
add a TS type to `html` that is a `string`

Reply:
created a PR in upstream
- https://github.com/vickonrails/next-starter-peacock/pull/25